exts/cesium.usd.plugins/schemas/cesium_schemas.usda

```diff
@@ -63,6 +63,47 @@ class CesiumDataPrim "CesiumDataPrim" (
         displayName = "Georeference Origin Height"
         doc = "The height of the origin in meters above the WGS84 ellipsoid. Do not confuse this with a geoid height or height above mean sea level, which can be tens of meters higher or lower depending on where in the world the origin is located."
     )
+
+    bool cesium:debug:disableMaterials = false (
+        customData = {
+            string apiName = "debugDisableMaterials"
+        }
+        displayName = "Disable Materials"
+        doc = "For debugging only: whether to disable materials."
```

Comment:
I'm not a fan of how these are written. To a native English speaker it will probably be okay, but to a non-native speaker I can see this adding confusion. Complete sentences such as, "Debugging value which disables materials when enabled," would be better.